web_src/js/features/repo-code.js

```diff
@@ -19,17 +20,18 @@ function selectRange($list, $select, $from) {
   $list.removeClass('active');
 
   // add hashchange to permalink
-  const $issue = $('a.ref-in-new-issue');
+  const $refInNewIssue = $('a.ref-in-new-issue');
   const $copyPermalink = $('a.copy-line-permalink');
   const $viewGitBlame = $('a.view_git_blame');
 
   const updateIssueHref = function (anchor) {
-    if ($issue.length === 0) {
+    if ($refInNewIssue.length === 0) {
       return;
     }
-    let href = $issue.attr('href');
-    href = `${href.replace(/%23L\d+$|%23L\d+-L\d+$/, '')}%23${anchor}`;
```

Comment:
What happened to this line anchor replacing?

Reply:
That's dirty code.

Reply:
I mean, I agree, but won't this mean that it is even copied to the `new issue` form?
Even if it doesn't have any effect there?

Reply:
The code has been rewritten. The `href` is newly constructed every time.

Reply:
Ah, I cleaned that code up myself mentally while reading:
I thought it would be applied on the anchor, not on the complete `href`.
In that case, it was even dirtier code than I imagined :sweat_smile: